reco_utils/azureml/wide_deep.py

```diff
@@ -34,7 +34,7 @@
 
 def _log(metric, value):
     if run is not None:
-        if type(value) == list and len(value) > 0 and (type(value[0]) == int or type(value[0]) == float):
+        if isintance(value, list) and len(value) > 0 and isinstance(value[0], (int, float))
             run.log_list(metric, value)
         else:
             run.log(metric, str(value))
```

Comment:
is it necessary to convert to string? the docs seem to imply that it's not (https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-track-experiments)

Reply:
fixed